app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -170,18 +172,20 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
     }
 
     private void resetPage() {
-        // forces reset here -> NECESSARY
-        adapter.update(showFilter.getAllShows());
+        // resets filters using same set up method
+        showFilter.setFilters();
```

Comment:
Let's make this `showFilter.resetFilters()` -- then we won't even need the comment you have above